libr/core/cmd_type.c

```diff
@@ -267,8 +267,22 @@ static void save_parsed_type(RCore *core, const char *parsed) {
 	}
 }
 
+static RCore *core_ = NULL; // HACK
+
 static int stdifstruct(void *user, const char *k, const char *v) {
-	return !strncmp (v, "struct", strlen ("struct") + 1);
+	if (!strncmp (v, "struct", strlen ("struct") + 1)) {
+		return true;
+	} else if (!strcmp (v, "typedef")) {
+		r_return_val_if_fail (core_ && core_->anal, false);
```

Comment:
those statements should be at the begining of the function not here

Reply:
done 466a38fc1987c892091597bff34a65ab5e60d6a2